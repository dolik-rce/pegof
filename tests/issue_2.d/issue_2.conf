ast
input issue_2.d/grammar.peg
