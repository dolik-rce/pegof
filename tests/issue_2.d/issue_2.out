GRAMMAR:
  COMMENT: "action can be anywhere in the rule, even at the very begining" line=1, col=3
  RULE:
    RULE_NAME: "X" line=3, col=1
    ALTERNATION:
      REFNAME: "A" line=3, col=6
      REFNAME: "B" line=3, col=10
      REFNAME: "C" line=3, col=14
  RULE:
    RULE_NAME: "A" line=4, col=1
    SEQUENCE:
      SOURCE: " printf("A!\n"); " line=4, col=7
      STRING: "A" line=4, col=27
  RULE:
    RULE_NAME: "B" line=5, col=1
    SEQUENCE:
      STRING: "B" line=5, col=7
      SOURCE: " printf("B!\n"); " line=5, col=11
  RULE:
    RULE_NAME: "C" line=6, col=1
    SEQUENCE:
      STRING: "C" line=6, col=7
      SOURCE: " printf("C!\n"); " line=6, col=11
      STRING: "C" line=6, col=31
